File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        String[] arr = my_string.split("");
        
        for(int i = 0; i < arr.length; i++){
            if(arr[i].charAt(0) >= 'a' && arr[i].charAt(0) <= 'z'){
                arr[i] = arr[i].toUpperCase();
            }else if(arr[i].charAt(0) >= 'A' && arr[i].charAt(0) <= 'Z'){
                arr[i] = arr[i].toLowerCase();
            }
        }
        return String.join("",arr);
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.js
function solution(my_string) {
    let arr = my_string.split("");
    
    for(let i = 0; i <arr.length; i++){
        if(arr[i] >= 'a' && arr[i] <= 'z'){
            arr[i] = arr[i].toUpperCase();
        }else
            arr[i] = arr[i].toLowerCase();
    }
    return arr.join("");
    
}